hyp3lib/get_orb.py

```diff
@@ -16,7 +17,43 @@
 from hyp3lib import OrbitDownloadError
 from hyp3lib.fetch import download_file
 
-ESA_AUTH = ('gnssguest', 'gnssguest')
+ESA_CREATE_TOKEN_URL = 'https://identity.dataspace.copernicus.eu/auth/realms/CDSE/protocol/openid-connect/token'
+ESA_DELETE_TOKEN_URL = 'https://identity.dataspace.copernicus.eu/auth/realms/CDSE/account/sessions'
+
+
+class EsaToken:
```

Comment:
I like this context manager. I think there may be other scenarios in the future where it could be useful.